sktime/forecasting/deep_learning/base.py

```diff
@@ -58,5 +58,81 @@ def _fit(self, y, X, fh):
 
     def _predict(self, X, **kwargs):
         """Predict with fitted model."""
-        self.network.eval()
-        return self.network(X)
+        dataloader = self.build_pytorch_dataloader(X)
+        y_pred = []
+        for x, _ in dataloader:
+            y_pred.append(self.network(x).detach().numpy())
+        return np.concatenate(y_pred)
```

Comment:
This creates 3D np.array of shape [n, pred_len, in_channels]. When this return is passed to `convert_to` in the internal `predict` of `BaseForecaster`, the error is thrown.

Reply:
would like to see the message, that should not happen?

I assume it happens in the default contract tests, so if you would kindly fix the linting issues, this should be visible.